README.md

```diff
@@ -1,26 +1,124 @@
-# Accessing Documentation
+# Quality of metadata with MetaQS backend service
 
-Documentation is generated with Sphinx and uploaded to [Confluence](https://confluence.edu-sharing.net/confluence/display/OEDE/Technische+Dokumentation)
+The _MetaQS_ service provides an API to calculate different statistics and metrics for the metadata.
+It uses Elasticsearch queries to achieve this.
 
-Go to `src` and install poetry dependencies.
+## Feature Documentation
 
-Start poetry shell.
+The documentation regarding how quality metrics are determined and the development setup is done can be found
+[here](./docs/index.rst).
 
-Go to `docs` and choose which build you want:
-
-## Local
+It can be converted to markdown with the following command:
 
 ```bash
-sphinx-build . build
+sphinx-build -M markdown ./ build_md
 ```
 
-Launch `index.html` in `build` directory
+## Local Setup
+
+MetaQS requires python 3.9 or later and [poetry](https://python-poetry.org/docs/#installation).
 
-## For confluence
+Setup details are explained [here](./docs/setup.rst).
 
-WIP: Currently not possible to automatically push to confluence.
-Add token from JIRA.
+## Code Style & Pre-commit hook
+
+The code style is enforced via tools configured in [.pre-commit-config.yaml](./src/.pre-commit-config.yaml) and
+[pyproject.toml](./src/pyproject.toml). After setting up the environment (which will install the python
+`pre-commit` script) one should install the hook via:
 
 ```bash
-./build_confluence.sh
+pre-commit install
 ```
+
+This will make sure that the commits follow the defined codestyle and the respective CI pipeline step will pass.
+
+## Unit-Tests
+
+Unit-tests are placed [here](./tests).
+
+All tests make use of mocks and static (checked in) resources. Hence, they can run independently of any docker
+container or other environment constraints.
+
+## Branching Model
+
+This repository follows the [GitHub Flow](http://scottchacon.com/2011/08/31/github-flow.html) branching model, but adds
```

Comment:
I would just reference Git-Flow (https://www.atlassian.com/de/git/tutorials/comparing-workflows/gitflow-workflow) then. Git-Flow inherently comes with the develop branch -  very much simplified: "GitHub-FLow = Git-Flow - develop branch"

Reply:
Done!